engine/src/dsklnx.cpp

```diff
@@ -995,6 +995,12 @@ class MCLinuxDesktop: public MCSystemInterface
             return MCStringFormat(r_folder, "%@/Desktop", *t_home);
         else if (MCNameIsEqualToCaseless(p_type, MCN_home))
             return MCStringCopy(*t_home, r_folder);
+        // 2019-12-19 MDW [[ feature_linux_documents_folder ]] allow linux to have "documents" == ~/Documents
+        else if (MCNameIsEqualToCaseless(p_type, MCN_documents))
+            return MCStringFormat(r_folder, "%@/Documents", *t_home);
+        // 2019-12-19 MDW [[ feature_linux_documents_folder ]] allow linux to have "system" == /opt
```

Comment:
I agree with adding support for documents like this - it is no more or less correct than the way we compute the desktop folder after all.

I don't agree with `/opt` being `system` though - given the filesystem hierarchy definition of 'opt' is for "the installation of add-on application software packages". It doesn't seem like a good fit.

Reply:
Looking at this, I agree about /opt. What do you think of the suggestion in https://quality.livecode.com/show_bug.cgi?id=21433 to use /sbin? That seems more reasonable to me.

Reply:
Heh - no - Richard gives no use-case nor any indication why /sbin should be used (isn't /sbin in $PATH?).

I can't see how /sbin is anything like /System on macOS, nor is it really much like the Windows folder (as most 'system binaries' I think are actually in a subfolder?). 

Its a separate issue anyway - if someone can come up with a convincing argument as to *why* that specialFolderPath token is useful (on Linux) and by doing so demonstrate that /sbin is the thing to use then I'll happily entertain a patch :)